content/en/docs/setup/additional-setup/customize-installation-helm/index.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Advanced Helm Chart Customization
+description: Describes how to customize installation configuration options when installing with helm.
+weight: 55
+keywords: [profiles,install,helm]
+owner: istio/wg-environments-maintainers
+test: n/a
+---
+
+## Prerequisites
+
+Before you begin, check the following prerequisites:
+
+1. [Download the Istio release](/docs/setup/getting-started/#download).
+1. Perform any necessary [platform-specific setup](/docs/setup/platform-setup/).
+1. Check the [Requirements for Pods and Services](/docs/ops/deployment/requirements/).
+1. [Usage of helm for Istio installation](/docs/setup/install/helm).
+1. Helm version that supports post rendering. (>= 3.1)
+1. kubectl or kustomize.
+
+## Advanced Helm Chart Customization
+
+Istio's helm chart tries to incorporate most of the attributes needed by users for their specific requirements. However, it does not
+contain every possible Kubernetes value you may want to tweak. While it is not practical to have such a mechanism in place, in this
+document we will demonstrate a method which would allow you to do some advanced helm chart customization without the need to directly
+modify Istio's helm chart.
+
+### Using Helm with kustomize to post-render Istio charts
+
+Using Helm post rendering capability, you can tweak the installation manifests to meet your requirements easily.
```

Comment:
I notice later we used `post-rendering`, so debating if we should use the back-ticks through the document.
```suggestion
Using the Helm `post-renderer`, you can tweak the installation manifests to meet your requirements easily.
```